x-pack/test/security_solution_cypress/serverless_config.ts

```diff
@@ -34,6 +34,9 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
           { product_line: 'endpoint', product_tier: 'complete' },
           { product_line: 'cloud', product_tier: 'complete' },
         ])}`,
+        `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+          'alertSuppressionForThresholdRuleEnabled',
```

Comment:
For serverless specific it is better to pass this value directly to the test and not in the config. This configuration is just used for the serverless FTR, so it is not going to work in a real serverless environment. When working with MKI we are reading all the information from the test :)

Reply:
added directly to tests